### DISCUSSION TITLE
Account restricted for violation of terms of service (Need Help)

### DISCUSSION THREAD

**Discussion by Poughe** [codewars/codewars.com#3206] [Q&A]
Hi,

My name is Ernest and I happened to get back into my codewars account and just happened to see the 'Account restricted for violation of terms of service.' I haven't been on the platform in years and was wondering why I have this restriction. This wasn't anything I have seen before when I was using the platform

https://www.codewars.com/users/Poughe

**Comment by hobovsky** [comment]
If you are still interested why the account has been restricted, you can write an email to info@codewars.com . This is preferred way of handling matters related to accounts now.
